packages/safe-ds-eda/types/state.d.ts

```diff
@@ -148,11 +151,18 @@ export interface ValueRangeFilter extends ColumnFilterBase {
 
 export interface SpecificValueFilter extends ColumnFilterBase {
     type: 'specificValue';
-    value: number;
+    value: string;
 }
 
-export type NumericalFilter = ValueRangeFilter | SpecificValueFilter;
-export type CategoricalFilter = SearchStringFilter;
+export interface PossibleSpecificValueFilter extends ColumnFilterBase {
+    type: 'specificValue';
+    values: string[];
+}
+
+export type NumericalFilter = ValueRangeFilter;
+export type CategoricalFilter = SearchStringFilter | SpecificValueFilter;
+
+export type PossibleColumnFilter = ValueRangeFilter | PossibleSearchStringFilter | PossibleSpecificValueFilter;
```

Comment:
* For consistency, rename this to `ColumnFilter`. 
* Also, the interface `ColumnFilterBase` probably needn't be exported anymore.

Reply:
This specifically defines possible filters though that I pass to my filter component and then based on this show those filters that are possible. Like `SearchStringFilter `defines the string searched for, while `PossibleSearchStringFilter `does not, and `ValueRangeFilter `defines the selected current min and max, while `PossibleValueRangeFilter `only defines the total possible min and max.
`ColumnFilter` is in theory `NumericalFilter | CategoricalFilter` but I use those directly depending on the column type and never so far unified.